src/util/ChartUtils.ts

```diff
@@ -1251,8 +1254,9 @@ export const parseSpecifiedDomain = (specifiedDomain: any, dataDomain: any, allo
  * @param  {Boolean} isBar if items in axis are bars
  * @return {Number} Size
  */
+type CartesianAxisProps = XAxisProps | YAxisProps | ZAxisProps;
```

Comment:
the jsdoc belongs to the function not the type, please move the type above the jsdoc comment